.redhat-ci.yml

```diff
@@ -0,0 +1,62 @@
+---
+
+# Example format:
+# https://github.com/jlebon/redhat-ci/blob/master/sample.redhat-ci.yml
+
+# Internal Job (for realtime monitoring):
+# https://hurl.corp.redhat.com/eb0faa3
+
+container:
+    image: docker.io/stackbrew/centos:latest
```

Comment:
Just going off of what [the page](https://hub.docker.com/r/stackbrew/centos/) says (I hadn't heard of stackbrew before):

> Deprecated; use 'centos' from https://hub.docker.com/_/centos/ instead

The `centos` image is the official one. I'm guessing the stackbrew one was available before `centos` existed?